editor/src/messages/portfolio/document/node_graph/document_node_types.rs

```diff
@@ -266,13 +266,14 @@ fn static_nodes() -> Vec<DocumentNodeDefinition> {
 							name: "To Artboard".to_string(),
 							manual_composition: Some(concrete!(Footprint)),
 							inputs: vec![
-								NodeInput::network(concrete!(graphene_core::GraphicGroup), 1),
+								NodeInput::network(concrete!(TaggedValue), 1),
+								NodeInput::value(TaggedValue::String(String::from("Artboard")), false),
```

Comment:
replace this with a comment like "NodeInput::Value is inserted at compile time based on document node name"